src/content/docs/mlops/integrations/openai-integration.mdx

```diff
@@ -4,58 +4,65 @@ metaDescription: "A lightweight tool to monitor your OpenAI workload."
 freshnessValidatedDate: never
 ---
 
-import modelperformancemonitoringOpenAIDashboard from 'images/model-performance-monitoring_screenshot-crop_openai-dashboard.webp'
+import modelperformancemonitoringOpenAIDashboard from 'images/model-performance-monitoring_screenshot-crop_openai-dashboard-1.webp'
 
-With the GPT series application integration, users have the ability to seamlessly monitor OpenAI completion queries and simultaneously log useful statistics in a New Relic customizable dashboard.
+With New Relic's GPT series application integration, you have the ability to seamlessly monitor all OpenAI queries. By adding just two lines of code, your team can gain access to key performance metrics such as cost, response time, rate limits, error rates, chat history, and sample inputs and outputs. Our GPT integration provides real-time metrics to help businesses optimize usage, reduce costs, and achieve better results.
```

Comment:
I'd only use OpenAI integration rather than GPT app integration, which is hard to parse. Also afaik we don't use New Relic in the possessive for elusive legal reasons